app/javascript/App.js

```diff
@@ -0,0 +1,21 @@
+import React, { useState, useEffect } from "react";
+import { BrowserRouter as Router, Route } from "react-router-dom";
+import Greeting from "./components/Greeting";
+
+function App() {
+  const [greeting, setGreeting] = useState("");
+
+  useEffect(() => {
+    fetch("/random_greeting")
+      .then(response => response.json())
+      .then(data => setGreeting(data.greeting));
+  }, []);
```

Comment:
- [ ] Kindly remember to use the redux for the global state and not the local state of React.

Reply:
Suggested changes have been implemented. Screen shot attached below for your reference.

![image](https://github.com/baqar-abbas/hello-rails-react1/assets/117091923/e1ce1546-42b9-4f5b-9785-ffcfc0139916)

Reply:
![image](https://github.com/baqar-abbas/hello-rails-react1/assets/117091923/8632b7be-91b8-49ba-bad4-a409f5583a11)

Reply:
![image](https://github.com/baqar-abbas/hello-rails-react1/assets/117091923/2e902ee4-489d-499e-b46c-3ed73212177b)